modules/user/user.graphql.js

```diff
@@ -26,7 +26,7 @@ input UserInput {
     lastName: String
     password: String
     role: String
-    email: String
+    email: String!
```

Comment:
undo this change, if the user will log in form facebook - he should be able to register

Reply:
We discussed it yesterday and discarded an option of no email, didn't we?..

Reply:
Done